app/components/common/__tests__/BaseInstructionCard.spec.tsx

```diff
@@ -13,7 +14,8 @@ describe('BaseInstructionCard', () => {
     test('should render "BaseInstructionCard"', async () => {
         const index = 1;
         const m = mock.deserializeMessageV0(stubs.aTokenCreateIdempotentMsg);
-        const connection = new Connection(clusterApiUrl('mainnet-beta'));
+        const realConnection = createRealConnection('mainnet-beta');
+        const connection = createMockConnection('common-base-instruction-card', realConnection);
```

Comment:
What do you think we change drift from the <%test-case-name%> to <%epoch_name%> to create fixtures?
In case we have two different tests created at the same time, they will produce separate fixtures. Correct me if I am wrong here.
That moves the focus from the current cluster state (epoch, for example) to the component and produces extra fixtures likely.